src/content/docs/tips-misc/presence.mdx

```diff
@@ -0,0 +1,100 @@
+
+```ts
+import { Client, ActivityType } from "seyfert";
+
+const client = new Client({
+    presence: (shardId) => ({
+        status: "idle",
+        activities: [{
+            name: "your servers",
+            type: ActivityType.Watching,
+        }],
+    })
+});
+```
+
+#### Explanation:
+- **`status`**: Defines the bot’s online status. Possible values are `online`, `idle`, `dnd` (Do Not Disturb), or `invisible`.
+- **`activities`**: An array of activities the bot is engaged in. Each activity requires:
+  - **`name`**: The custom status message.
+  - **`type`**: The activity type, such as `Playing`, `Listening`, `Watching`, etc. These types are constants in the `ActivityType` enum.
+
+### Rendering the Bot as "Mobile"
+
+To display your bot as if it’s running on a mobile device, you can specify custom gateway properties. Here’s how:
+
+```ts
+const client = new Client({
+    gateway: {
+        properties: {
+            os: 'android',
+            browser: 'Discord Android',
+            device: 'android'
+        }
+    },
+    presence: (shardId) => ({
+        // status: "idle" - Leave blank
+        activities: [{
+            name: "your servers",
+            type: ActivityType.Watching,
+        }],
+    })
+});
+```
+
+#### Important Notes:
+- **`gateway.properties`**: Defines the operating system, browser, and device to emulate. When set to mimic a mobile device (e.g., `os: 'android'`), the bot will appear with a mobile icon.
+- **Avoiding `status`**: If you set the `status` property explicitly, the mobile icon will not be rendered. For example, leaving `status` undefined allows the mobile icon to appear.
+
+### Dynamically Updating Presence
+
+Once the bot is running, you can dynamically update its presence by creating a command to handle presence updates. This is particularly useful when the update is triggered by user interaction or other events:
+
+```ts
+import { Command, ActivityType, PresenceUpdateStatus } from "seyfert";
+
+@Declare({
+    name: 'presence',
+    description: 'Change bot presence'
+})
+export default class PresenceCommand extends Command {
+    async run(ctx: GuildCommandContext) {
+        ctx.client.gateway.setPresence({
+            activities: [{
+                name: "with Seyfert",
+                type: ActivityType.Playing,
+            }],
+            status: PresenceUpdateStatus.Online,
+            since: Date.now(),
+            afk: false
+        });
+
+        await ctx.editOrReply({ content: 'Switched Presence' });
+    }
+}
+```
+
+#### Explanation:
+- **Command Definition**: The `@Declare` decorator defines a command called `presence` to handle presence updates.
+- **`setPresence` Method**: Updates the bot’s activities, status, and additional details dynamically.
+- **User Feedback**: After updating the presence, the bot confirms the action by sending a message.
+
+### Best Practices
```

Comment:
It would be great if you could add the fact that the gateway has a ratelimit and although it is wide, it is advisable not to change the presence too constantly.